projects/angular/src/data/datagrid/utils/key-navigation-grid.controller.ts

```diff
@@ -88,6 +89,12 @@ export class KeyNavigationGridController implements OnDestroy {
           }
         });
 
+      fromEvent(this.grid, 'wheel')
+        .pipe(takeUntil(this.destroy$))
+        .subscribe(() => {
+          this.removeActiveCell();
+        });
+
```

Comment:
Why do we need that? It's removing the focus if you scroll with the mouse couple of times and the row is still in view.

Reply:
It will be triggered only after data update. If the data is not changed the focus will remain.

Reply:
It's tracking 'wheel' events so it's being removed if you focus some cell and start scrolling -- check video
https://github.com/vmware-clarity/ng-clarity/assets/127101685/6613cb6a-734a-4580-ac66-a9fd726bdb65

Reply:
This issue will be addressed in [CDE-2043](https://jira.eng.vmware.com/browse/CDE-2043)